README.md

```diff
@@ -25,7 +25,7 @@ And then execute:
 Once the installation is done you can use Pecorino to start defining your throttles. Imagine you have a resource called `vault` and you want to limit the number of updates to it to 5 per second. To achieve that, instantiate a new `Throttle` in your controller or job code, and then trigger it using `Throttle#request!`. A call to `request!` registers 1 token getting added to the bucket. If the bucket is full, or the throttle is currently in "block" mode (has recently been triggered), a `Pecorino::Throttle::Throttled` exception will be raised.
 
 ```ruby
-throttle = Pecorino::Throttle.new(key: "vault", leak_rate: 5, capacity: 5)
+throttle = Pecorino::Throttle.new(key: "vault", over_time: 1, capacity: 5)
```

Comment:
We should probably make it clear if over_time should be an integer representing seconds or a Duration.
If it is a duration then this example could be over_time: 1.seconds
We could also verify in the initializer that a duration is passed for the argument.
Otherwise the reader has to go check the initializer and see that it does a to_f etc

Reply:
`over_time` will work as a parameter name, but it is a good idea to document that it will accept _either_ a `Duration` or a number of seconds and convert it to float.